src/Parameters/CreateMeetingParameters.php

```diff
@@ -1074,6 +1108,8 @@ public function getHTTPQuery()
             'lockSettingsLockOnJoin'             => $this->isLockSettingsLockOnJoin() ? 'true' : 'false',
             'lockSettingsLockOnJoinConfigurable' => $this->isLockSettingsLockOnJoinConfigurable() ? 'true' : 'false',
             'allowModsToUnmuteUsers'             => $this->isAllowModsToUnmuteUsers() ? 'true' : 'false',
+            'bannerText'                         => $this->bannerText,
```

Comment:
@sualko  better to used method to get a value here instead of variable in both bannerText and bannerColor

i am ok with this pr.